soc/riscv/riscv-ite/it8xxx2/soc.c

```diff
@@ -278,6 +278,17 @@ static int ite_it8xxx2_init(void)
 	 */
 	gctrl_regs->GCTRL_WMCR |= BIT(7);
 
+	/*
+	 * Disable this feature that can detect pre-define hardware
+	 * slave A through I2C0. This is for debugging use, so it
```

Comment:
Please use "target" instead of "slave".  https://github.com/zephyrproject-rtos/zephyr/issues/27033

Reply:
Done.